vir/defs/polymorphic/ast/expr.rs

```diff
@@ -617,7 +633,10 @@ impl Expr {
 
     pub fn is_only_permissions(&self) -> bool {
         match self {
-            Expr::PredicateAccessPredicate(..) | Expr::FieldAccessPredicate(..) => true,
+            Expr::PredicateAccessPredicate(..)
+            // TODO check this not sure
```

Comment:
Assuming this is used for e.g. purification, then setting this to `true` looks correct. For now we are using resources to verify time anyway.